src/core/frame.h

```diff
@@ -253,6 +255,39 @@ QuicResetStreamFrameDecode(
     _Out_ QUIC_RESET_STREAM_EX* Frame
     );
 
+//
+// QUIC_FRAME_RELIABLE_RESET_STREAM Encoding/Decoding
+//
+typedef struct QUIC_RELIABLE_RESET_STREAM_EX {
+
+    QUIC_VAR_INT StreamID;
+    QUIC_VAR_INT ErrorCode;
+    QUIC_VAR_INT FinalSize;
+    QUIC_VAR_INT ReliableSize;
+    
+} QUIC_RELIABLE_RESET_STREAM_EX;
```

Comment:
Let's add a test to FrameTest.cpp for this new frame.

Reply:
Added! Will update.